README.md

```diff
@@ -1,2 +1,49 @@
-# Spiral-Tech-Project
-The module one capstone project at microverse.
+#Tech in Africa (TiA Summit) Webpage
+
+# Description of the project.
+
+The project is a webpage design personalised version of the module one capstone project guide. It has been designed for a non-real summit tagged the Tech in Africa (TiA) hosted to highlight the achievemnts and growth of young African tech community through innovation. The summit is to among other things create avenue for tech community to interact and showcase latest tech trends and innovation in Africa's tech community as well as the challenges.
+
+Built With:
+-HTML
+
+- CSS
+
+- ESLint
+
+#Live Demo
+[Live Demo Link] (https://gids-dadi.github.io/The-portfolio-project/)
+
+#Link to Video
+https://www.loom.com/share/e30b4f34fca341d9a36e054e26aaf715
+
+
+## Getting Started
+
+-Clone this repository to your desktop.
+-Navigate to the top level of the directory.
+-Open index.html in your browser to view the template.
+-click on the menu to navigate to about page.
+
+👤 Gideon Dadi
+
+-Linkedin [@GideonDadi] (https://www.linkedin.com/feed/)
+
+- GitHub: [@gids_dadi](https://github.com/gids-dadi)
+- Twitter: [@Dadi_AG](https://twitter.com/Dadi_AG)
+
+Show your support
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+-All the collaborators on this branch.
```

Comment:
- [ ] Please kindly give credit to the creator of the design `Cindy Shin` in the acknowledgements section, and also include a link to the design.